### PR TITLE
adding feature 123

### PR DESCRIPTION
**Pull request by thisIsOwais**
new feature work done for ticket 123

### CODE REVIEW

File: customer/src/services/customer-service.test.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // which service it is
 describe("CustomerService", () => {
-  // This will explain which function we are testing for customer service!
+  // This will explain which function we are testing for services of customer!
```
Comment (iammohd) [inline customer/src/services/customer-service.test.js]: do you think it is required here

Comment (thisIsOwais) [reply customer/src/services/customer-service.test.js]: please approve this

Comment (iammohd) [reply customer/src/services/customer-service.test.js]: looks great!

File: customer/src/services/customer-service.test.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // which service it is
 describe("CustomerService", () => {
-  // This will explain which function we are testing for customer service!
+  // This will explain the functionality we are doing for customer services!
```
Comment (iammohd) [inline customer/src/services/customer-service.test.js:3]: looks great

**Review by iammohd — APPROVED** [review]
great work!

File: customer/src/services/customer-service.test.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // which service it is
 describe("CustomerService", () => {
-  // This will explain which function we are testing for customer service!
+  // This will explain which function we are testing for services of customer!
   describe("SignIn", () => {
     // Which Scenario we are testing
     test("validate user inputs", () => {});
```
Comment (iammohd) [inline customer/src/services/customer-service.test.js:6]: great

**Review by iammohd — APPROVED** [review]

**Review by iammohd — APPROVED** [review]

**Pull request merged**